doc/manual/install-domserver.rst

```diff
@@ -179,6 +179,22 @@ this approach is that the webserver is not aware of the actual client IP. This
 means that access logs for the webserver will still report the IP of the proxy
 or loadbalancer.
 
+Scaling the domserver
+---------------------
+
+Documenting all possible ways to properly size the domserver is out of scope for this
+manual. But properly think about the memory management for the PHP-FPM children, we advice
+40 FPM children per gigabyte of RAM memory so around 500 for a 16Gb system.
+
+* Consider the amount of submissions you expect at any given time, does the load stay uniform?
+* Do you expect all teams to solve the easy problems in the first hour?
+  As a rule of thumb you can expect: ``#testcases * #submissions/teams`` as extra load (HTTP traffic
+  from the judgehost reporting the results per testcase additionally on the team refreshing the page
+  to know the verdict). Similar for the harder problems (which look simple) where multiple wrong
```

Comment:
This is really hard to understand. I'm also not sure why dividing by the number of teams?
Maybe something like:
```suggestion
    Do not forget to account for "hidden load", i.e. requests that the judging system makes to the domserver. You can expect ``#testcases * #submissions/teams`` **requests/sec (or what unit is this?)** that you need to account for. Additionally, the scoreboard pages for spectators and teams auto-refresh every 30s (**not sure if this is still true, but I suspect it is**), so every 30 teams = 1 req/sec minimum (on top of teams refreshing manually to know the verdict from their submission).
```

I'm not sure how much to write here, but lazy evaluation is a big factor too.

Reply:
the `/`was not meant for dividing but more a `or` at this point the number of submissions is equal to the number of teams (1 per team for the simple one)

I fully agree that your way of reading it makes more sense as it mixes up 2 uses for the `/` symbol.

Reply:
I agree with @nickygerritsen, dropping `/teams` makes it more legible here. To iterate on ubergeek's suggestion

```suggestion
Do not forget to account for "hidden load", i.e. requests that the judging system makes to the domserver. You can expect ``#testcases * #submissions`` requests that you need to account for. Not only are these requests spread out over the duration of the contest but they can also be spread out unevenly. The 'middle three' hours of a full five hour contest usually see fewer submissions than the first and last hours. 

This "hidden load" can be mitigated by keeping the "lazy evaluation" option enabled. Lazy evaluation stops whenever the judgehost decides a verdict can be reached. e.g. a submission who's verdict on the first sample testcase is "Wrong Answer" does not need to be checked against the other testcases. Lazy evaluation helps when it is expected that many submissions fail which is often the case.

Additionally, the scoreboard pages for spectators and teams auto-refresh every 30s, so every 30 teams = 1 req/sec minimum (on top of teams refreshing manually to know the verdict from their submission).
```